src/flux/task-plan.coffee

```diff
@@ -168,6 +172,12 @@ TaskPlanConfig =
     tasking_plans = @_getDefaultTaskingTimes(id, courseId, periods)
     @_change(id, {tasking_plans})
 
+  # set taskings that lack an opens_at to the default
+  setBlankOpensAtDateForCourse: (id, courseId, opensAtDate) ->
```

Comment:
yeah, that was an initial stab at setting the date, then I noticed the stuff in the component was doing it